Client QA items

nahbee10 opened the pull request:
### Description

<!--- Summarize the changes that can be found in this PR and how your reviewers should test these changes. -->
Fixed these items from the client QA board on Notion([link](https://www.notion.so/garden3d/142131fea2c78138bc87eab60d594492?v=142131fea2c7814b9724000cdb974c5f&pvs=4), all the items under the 'in progress' column)
- Change “Ecosystem” to “Protocol” in footer
- Line height of menu items in side bar is too tall
- Header displaying weird scroll bar 
- Grey scrollboxes around side bar
- Link text isn’t wrapping to the next line properly
- Add hover states to featured CTAs
- Remove ‘Connect with us’ section from Home page 
- In article pages, collapse parent section in side bar by default
- Suggested search dropdown extends past search bar
- Slightly increase line height of text in articles

### Type(s) of changes

<!--- Put an `x` in all the boxes that apply. -->

- [x] Bug fix
- [ ] New feature
- [x] Update to an existing feature

### Motivation for PR

<!--- If it addresses an open issue, please link to the issue here, otherwise, briefly describe the issue. -->
Client QA items under the 'in progress' column([link](https://www.notion.so/garden3d/142131fea2c78138bc87eab60d594492?v=142131fea2c7814b9724000cdb974c5f&pvs=4))

### How Has This Been Tested?

<!--- Please note how you have tested your changes. Browsers, accessibility, devices, unit tests, etc. -->
tested on local and production preview

**HOW TO PREVIEW:
Step 1. Login to Uniswap Labs Zendesk 

Step 2. Open this link(https://support.uniswap.org/theming/theme/65eaf0e7-4b24-40f6-bdf1-34d5c46d85a2) to activate the staging theme(will not affect the production theme)

Step 3. Open this link(https://support.uniswap.org/hc/en-us) to view the staging site without iframe nest

### Applicable screenshots

<!--- When appropriate, upload screenshots. -->

### Follow-up PR

<!--- When appropriate, please note what your reviewers can expect in a follow up PR. -->

mokaymokay reviewed (changes requested):
See comments: https://www.loom.com/share/6e251c479cd84ff0b8b427b0b58a1887

nahbee10 commented:
> See comments: 

@mokaymokay addressed the comments in the preview(use the same way above to preview it!)

1. fixed the font size of <li>
<img width="596" alt="Screenshot 2024-12-10 at 8 35 02 PM" src="https://github.com/user-attachments/assets/fbf2d609-395f-47aa-b23a-689539f2b173">

2. added bottom padding to the heading texts:
<img width="718" alt="Screenshot 2024-12-10 at 8 09 58 PM" src="https://github.com/user-attachments/assets/c69795a9-fb39-414f-a7f8-839174c83362">

3. removed hover state from chevron icon on the right side of side nav items:
<img width="222" alt="Screenshot 2024-12-10 at 8 34 39 PM" src="https://github.com/user-attachments/assets/0ecd8d3b-5e1f-4379-8778-d52e6a9db844">

4. applied shade to the right side of overflowing breadcrumb([Figma design](https://www.figma.com/design/QkBERlCkBKBAWRqELCC8uq/%5BUniswap%5D-%F0%9F%8D%84-Spore-Library-w%2F-Web?m=auto&node-id=5083-15456))
<img width="497" alt="Screenshot 2024-12-10 at 8 38 24 PM" src="https://github.com/user-attachments/assets/4627d053-e511-43d8-afa7-1dcb014a1c94">

5. align autocomplete search result window to the search text field width:
<img width="1164" alt="Screenshot 2024-12-10 at 8 23 33 PM" src="https://github.com/user-attachments/assets/b4f2d221-a44d-43cf-b01a-3e43602b1a1f">

mokaymokay reviewed (approved):
Noticing one more thing!

If I'm on a category page, looks like the parent is highlighted but the category isn't highlighted on mobile/tablet nav. 
<img width="1198" alt="Screenshot 2024-12-10 at 2 43 13 PM" src="https://github.com/user-attachments/assets/28de68ac-a29b-4a1e-bc2a-eb7b2f47f3ed">

See intended behavior on desktop
<img width="1663" alt="Screenshot 2024-12-10 at 2 44 02 PM" src="https://github.com/user-attachments/assets/d51d2c40-1642-4da1-ab1e-9eef43bd8451">

mokaymokay commented on src/modules/side-nav/MobileMenuNavModule.tsx:
```diff
@@ -96,6 +96,7 @@ export const MobileMenuNav: FC<Props> = ({ sideNavData, navState }) => {
           <ul className="accordion-body">
             {selectedCategory &&
               selectedCategory.sections.map((section) => {
+                console.log(section, navState.category, selectedCategory.id);
```
remove?
